cmd/server/main.go

```diff
@@ -301,6 +311,7 @@ func newProxyRunOptions() *ProxyRunOptions {
 		agentServiceAccount:       "",
 		kubeconfigPath:            "",
 		authenticationAudience:    "",
+		proxyStrategies:           "default",
```

Comment:
We should validate proxyStrategies (eg make it hasn't been configured to empty and the strategies specified are all recognized.)